server/tests/backends/fixtures/delete/non_existing_column.json

```diff
@@ -0,0 +1,55 @@
+{
+  "description": "it should not fail when deleting a column that does not exist in the dataset",
+  "exclude": [
+    "mysql"
+  ],
```

Comment:
This test fails on snowflake and postgres, maybe you'll want to exclude them too ? Or adapt them to mkae the behaviour consistent

Reply:
I'm gonna try to adapt them: if we can prevent a fail everywhere, let's do it!